execution/engine/execution_engine_test.go

```diff
@@ -269,7 +269,12 @@ func TestExecutionEngine_Execute(t *testing.T) {
 				return
 			}
 
-			assert.Equal(t, testCase.expectedResponse, actualResponse)
+			if testCase.expectedResponse == "" {
```

Comment:
It doesn't really matter but I can't help but think there is a way for this to always work without the if else

Reply:
Some tests put ` `` ` in the `expectedResponse` box, which is not valid JSON, like L1733, with `runWithAndCompareError` or the equivalent `run` invocation.

Reply:
I can just put this back as `Equal` as well, it doesn't matter really once you're done debugging.